docs/articles/documentation/test-api/selecting-page-elements/examples-of-working-with-dom-elements.md

```diff
@@ -81,7 +102,49 @@ test('My test', async t => {
 });
 ```
 
-## Checking if an Element is Available
+## Access Child Nodes in the DOM Hierarchy
+
+The selector API includes methods that allow you to [search for elements in the DOM tree](selectors/functional-style-selectors.md#search-for-elements-in-the-dom-hierarchy). For instance, you can use the [child](selectors/functional-style-selectors.md#child), [parent](selectors/functional-style-selectors.md#parent), and [sibling](selectors/functional-style-selectors.md#sibling) methods to traverse up, down or aside through the hierarchy.
+
+However, these methods have a limitation. They iterate DOM elements only and ignore other node types. To get a child text node for a selector, use a function as shown in this example.
```

Comment:
Actually we have different logic for different Selector's methods
searching for elements only:  
```
nth
withText
withExactText
withAttribute
filterVisible
filterHidden
filter (without using function or other Selector in constructor)

child
sibling
nextSibling
prevSibling
```
searching also for nodes:  
```
filter (e.g. Selector(() => document.body.childNodes).filter((node) => {});)

find
parent (Selector('body').parent((node) => {return node;}); can return document node)
```